src/getstream/updateBetterChannelMembers.js

```diff
@@ -37,6 +37,7 @@ const getAllChatAnonimityUserInfo = async (channelId, userIds = []) => {
   }
 };
 
+// eslint-disable-next-line no-underscore-dangle
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `eslint-disable-next-line no-underscore-dangle` is not recommended. It's better to follow the linting rules for consistency and readability across your codebase. If you need to disable a rule, it's better to do so in the ESLint configuration file for the entire project, rather than disabling it on a per-line basis.

```diff
- // eslint-disable-next-line no-underscore-dangle
  const __getAnonUserInfoPriority = (memberDataFromDb, member) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->